CMakeLists.txt

```diff
@@ -74,7 +74,7 @@ endif()
 
 include(FindPkgConfig)
 
-pkg_check_modules(BML REQUIRED bml)
+pkg_check_modules(BML bml REQUIRED)
```

Comment:
According to the [CMake documentation](https://cmake.org/cmake/help/v3.0/module/FindPkgConfig.html) the syntax is:

```cmake
pkg_check_modules(<PREFIX> [REQUIRED] [QUIET] <MODULE> [<MODULE>]*)
```